ngsa/src/main/java/com/cse/ngsa/app/utils/CommonUtils.java

```diff
@@ -19,6 +22,9 @@
 @Component
 public class CommonUtils {
 
+  @Autowired
+  ApplicationContext context;
```

Comment:
Since we are using BuildConfig.getBuildVersion(), we might as well use AutoWire BuildConfig (com.cse.ngsa.app.config.BuildConfig).

Thoughts?